src/App.js

```diff
@@ -11,6 +10,9 @@ function App() {
         <Route path="/" element={<Home />} />
         <Route path="/mock" element={<MockAPI />} />
         <Route path="/products" element={<Products />} />
+        <Route path="/cart" element={<Cart />} />
+        <Route path="/login" element={<Login />} />
```

Comment:
The cart should be a private route. Only the logged in users should be able to access it